### PR TITLE
219 Add form-validator

### LINKED ISSUE #219: Add form-validator

### PR DESCRIPTION
**Pull request by kaseyhinton**
closes #219

### CODE REVIEW

**Comment by aarondrabeck** [comment]
Looking good!  Please add a demo

File: packages/titanium-input-validator/package.json [hunk 1]
```diff
@@ -0,0 +1,38 @@
+{
+  "name": "@leavittsoftware/titanium-input-validator",
+  "description": "",
+  "author": "Leavitt Software",
+  "version": "3.6.0",
```
Comment (aarondrabeck) [inline packages/titanium-input-validator/package.json:5]:
```suggestion
  "version": "0.0.1",
```

File: packages/titanium-input-validator/src/titanium-input-validator.ts [hunk 2]
```diff
@@ -0,0 +1,56 @@
+import { css, html, LitElement } from 'lit';
+import { property, customElement, state } from 'lit/decorators.js';
+
+@customElement('titanium-input-validator')
+export class TitaniumInputValidatorElement extends LitElement {
+  @property({ type: String }) validationMessage = '';
+  @state() evaluator = () => true;
```
Comment (aarondrabeck) [inline packages/titanium-input-validator/src/titanium-input-validator.ts:7]:
```suggestion
  @property({...}) evaluator = () => true;
```

File: packages/titanium-input-validator/src/titanium-input-validator.ts [hunk 3]
```diff
@@ -0,0 +1,56 @@
+import { css, html, LitElement } from 'lit';
+import { property, customElement, state } from 'lit/decorators.js';
+
+@customElement('titanium-input-validator')
+export class TitaniumInputValidatorElement extends LitElement {
+  @property({ type: String }) validationMessage = '';
```
Comment (aarondrabeck) [inline packages/titanium-input-validator/src/titanium-input-validator.ts:6]: Add JS docs

Comment (kaseyhinton) [reply packages/titanium-input-validator/src/titanium-input-validator.ts:6]: Was going to do this in the follow up PR as noted but I can do it now

**Review by aarondrabeck — APPROVED** [review]

**Pull request merged**